.env.sample

```diff
@@ -107,4 +107,5 @@ PROMPTFLOW_ENDPOINT=
 PROMPTFLOW_API_KEY=
 PROMPTFLOW_RESPONSE_TIMEOUT=120
 PROMPTFLOW_REQUEST_FIELD_NAME=question
-PROMPTFLOW_RESPONSE_FIELD_NAME=answer
\ No newline at end of file
+PROMPTFLOW_RESPONSE_FIELD_NAME=answer
```

Comment:
The default is 'reply' in the code here, would it make sense to set the same default in the environment file?

Reply:
addressed, update default value for question field as well